frontend/src/App.js

```diff
@@ -1,29 +1,84 @@
 import React from "react";
 import "./App.scss";
 import "bootstrap/dist/css/bootstrap.min.css";
+import "@fortawesome/fontawesome-free/css/all.min.css";
+import createMuiTheme from "@material-ui/core/styles/createMuiTheme";
+import { MuiThemeProvider } from "@material-ui/core/styles";
 import { BrowserRouter, Route, Redirect, Switch } from "react-router-dom";
 import Register from "./components/Auth/Register/Register";
 import Login from "./components/Auth/Login/Login";
+import Menu from "./components/Menu/Menu";
 import GameView from "./components/GameView/GameView";
+import Settings from "./components/Settings/Settings";
+import ParticlesComponent from "./components/ParticlesComponent/ParticlesComponent";
+
+const theme = createMuiTheme({
+  overrides: {
+    MuiInputLabel: {
+      root: {
+        color: "white",
+        "&$focused": {
+          color: "white",
+          "&$error": {
+            color: "#f44336"
+          }
+        }
+      }
+    },
+    MuiInput: {
+      root: {
+        color: "white"
+      },
+      underline: {
+        "&:after": {
+          borderBottomColor: "white"
+        },
+        "&:before": {
+          borderBottomColor: "white"
+        },
+        "&:hover:not($disabled):not($focused):not($error):before": {
+          borderBottomColor: "white"
+        }
+      }
+    },
+    MuiButton: {
+      outlined: {
+        borderColor: "white",
+        color: "white",
+        "&:hover": {
+          backgroundColor: "white",
+          color: "black"
+        }
+      }
+    }
+  }
+});
 
 function App() {
   return (
     <div className="App">
-      <BrowserRouter>
-        <>
-          <Switch>
-            <Route exact path="/">
-              <Redirect to="register" />
-            </Route>
-            <Route path="/register" component={Register} />
-            <Route path="/login" component={Login} />
-            <Route path="/game" component={GameView} />
-            <Route path="*">
-              <Redirect to="register" />
-            </Route>
-          </Switch>
-        </>
-      </BrowserRouter>
+      <ParticlesComponent />
+      <div className="foreground">
+        <MuiThemeProvider theme={theme}>
+          <BrowserRouter>
+            <>
```

Comment:
Fragment here is unnecessary I think

Reply:
You told me to crate Fragment there. But ok I'll remove it.